bin/analytics

```diff
@@ -10,143 +10,82 @@ program :name, 'simulator.rb'
 program :version, '0.0.1'
 program :description, 'scripting simulator'
 
-# use an env var for write key, instead of a flag
-Analytics = Segment::Analytics.new({
-  write_key: ENV['SEGMENT_WRITE_KEY'],
-  on_error: Proc.new { |status, msg| print msg }
-})
-
 def toObject(str)
-  return JSON.parse(str)
-end
-
-# high level
-# analytics <method> [options]
-# SEGMENT_WRITE_KEY=<write_key> ./analytics.rb <method> [options]
-# SEGMENT_WRITE_KEY=<write_key> ./analytics.rb track --event testing --user 1234 --anonymous 567 --properties '{"hello": "goodbye"}' --context '{"slow":"poke"}'
-
-
-
-# track
-command :track do |c|
-  c.description = 'track a user event'
-  c.option '--user <id>', String, 'the user id to send the event as'
-  c.option '--event <event>', String, 'the event name to send with the event'
-  c.option '--anonymous <id>', String, 'the anonymous user id to send the event as'
-  c.option '--properties <data>', 'the event properties to send (JSON-encoded)'
-  c.option '--context <data>', 'additional context for the event (JSON-encoded)'
-
-  c.action do |args, options|
-    Analytics.track({
-      user_id: options.user,
-      event: options.event,
-      anonymous_id: options.anonymous,
-      properties: toObject(options.properties),
-      context: toObject(options.context)
-       })
-    Analytics.flush
+  if str
+    return JSON.parse(str)
   end
-
 end
 
-# page
-command :page do |c|
-  c.description = 'track a page view'
-  c.option '--user <id>', String, 'the user id to send the event as'
-  c.option '--anonymous <id>', String, 'the anonymous user id to send the event as'
-  c.option '--name <name>', String, 'the page name'
-  c.option '--properties <data>', 'the event properties to send (JSON-encoded)'
-  c.option '--context <data>', 'additional context for the event (JSON-encoded)'
-  c.option '--category <category>', 'the category of the page'
-  c.action do |args, options|
-    Analytics.page({
-      user_id: options.user,
-      anonymous_id: options.anonymous,
-      name: options.name,
-      properties: toObject(options.properties),
-      context: toObject(options.context),
-      category: options.category
-       })
-    Analytics.flush
-  end
+# analytics -method=<method> -segment-write-key=<segmentWriteKey> [options]
 
-end
+command :"" do |c|
```

Comment:
Nitpick: The common syntax for this is to do `command :something ... ` followed by `default_command :something`. [ref](https://github.com/commander-rb/commander#command-defaults)

Reply:
Updated!